### PR TITLE
units: Add a long name for Astronomical Unit

### PR DESCRIPTION
**Pull request by adrn**
This is a trivial pull request, but I noticed there is no "long name" for `u.AU` in the units package. Others may have opinions on how exactly to format the long name, but right now it is included as `u.AstronomicalUnit`.

__Currently:__

    >>> import astropy.units as u
    >>> u.au.long_names
    []

__This PR:__

    >>> u.au.long_names
    [u'AstronomicalUnit']

### CODE REVIEW

File: astropy/units/astrophys.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 ###########################################################################
 # LENGTH
 
-def_unit((['AU', 'au'], []), _si.au.value * si.m, namespace=_ns, prefixes=True,
+def_unit((['AU', 'au'], ['AstronomicalUnit']), _si.au.value * si.m, namespace=_ns, prefixes=True,
```
Comment (embray) [inline astropy/units/astrophys.py]: I'm for `astronomical_unit` myself.

**Comment by adrn** [comment]
@embray I'm fine with that too -- do you want to cancel the travis builds until we decide on a name? No sense wasting the compute time

**Comment by embray** [comment]
Eh, if it passes that's fine.  If you end up changing this you can also amend the commit message to include `[skip ci]`.

File: astropy/units/astrophys.py (hunk 1 above)
Comment (mdboom) [inline astropy/units/astrophys.py]: I'll throw in my vote for `astronomical_unit` as well, for the reason that most other units are lower case unless they are named after a person.

**Comment by adrn** [comment]
Ready for final review, I think?

**Comment by mhvk** [comment]
+1. Better with the underscore (though `nauticalmile` gives a counterexample, but I do think this is better).

**Comment by eteq** [comment]
:+1: 

I'll leave this for a bit in case someone else wants a look, but I'll milestone it for 1.0 as its pretty much set unless someone objects.

**Comment by astrofrog** [comment]
:+1:

**Pull request merged**